v1/src/main/java/com/google/cloud/teleport/spanner/ddl/Index.java

```diff
@@ -111,7 +116,9 @@ private void prettyPrintPg(Appendable appendable) throws IOException {
 
   private void prettyPrintGsql(Appendable appendable) throws IOException {
     appendable.append("CREATE");
-    if (unique()) {
+    if (type().equals("SEARCH")) {
+      appendable.append(" SEARCH");
```

Comment:
Yes, if the type is a search index, then the next token must be the keyword SEARCH, otherwise the existing logic checks if it's a unique index. Search indexes can never have the UNIQUE keyword so these cases are mutually exclusive.